### PR TITLE
Add the settings section

### PR DESCRIPTION
**Pull request by ptibulle**
Update the codelab.md file with a new section about Environment Settings

### CODE REVIEW

File: codelab.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 author: Maxime LUMEAU, Robin CAROFF and Pierre TIBULLE
 summary: Kotlin Multiplatform Workshop
-id: codelab-kotlin-Multiplatform
+id: codelab-kotlin-multiplatform
 categories: codelab,kotlin
 environments: Web
-status: Draft
+status: Published
```
Comment (RobinCaroff) [inline codelab.md]: @ptibulle Why do we have to set it as published ? It's a draft isn't it ?

File: codelab-kotlin-multiplatform/codelab.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "mainga": "UA-49880327-14",
+  "updated": "2019-08-12T16:25:44+02:00",
+  "id": "codelab-kotlin-multiplatform",
+  "duration": 6,
+  "title": "Workshop Kotlin Multiplatform",
+  "summary": "Kotlin Multiplatform Workshop",
+  "source": "codelab.md",
+  "theme": "",
+  "status": [
+    "published"
```
Comment (RobinCaroff) [inline codelab-kotlin-multiplatform/codelab.json]: @ptibulle Why is it set as published ?

File: codelab.md (hunk 1 above)
Comment (ptibulle) [reply codelab.md]: Yes it is ! Copy/Paste problem ???

Comment (RobinCaroff) [reply codelab.md]: OK. I'll change it for Draft then.

File: codelab-kotlin-multiplatform/codelab.json (hunk 2 above)
Comment (RobinCaroff) [reply codelab-kotlin-multiplatform/codelab.json]: I'll change it for draft.

**Pull request merged**